backend/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+db:
+  image: postgres:9.5
+web:
+  build: .
+  command: python manage.py runserver 0.0.0.0:8000
+  volumes:
+    - .:/opt/akvo-dash
```

Comment:
Yeah this is a very common approach to have a truly development environment. Let's standardise "/app" :smiley: